View Controllers/DetailsViewController.m

```diff
@@ -71,6 +77,38 @@ - (void)setPlace:(GMSPlace*)place {
     _place = place;
 }
 
+- (void)initUsersLabel {
+    [self.parsePlace getUsersCheckedInWithCompletion:^(NSArray<NSString *> * _Nullable users) {
+        self.usersCheckedIn = [[NSArray alloc] init];
+        
+        //init set of user objects from object ids; we only want unique users
+        NSMutableSet *mutableUsers = [NSMutableSet setWithArray:users];
+        NSString *myUserObjectId = PFUser.currentUser.objectId;
+        
+        //remove self from array
+        if ([mutableUsers containsObject:myUserObjectId]) {
+            [mutableUsers removeObject:myUserObjectId];
+        }
+        
+        //convert array of object IDs to array of PFUsers
+        [PFUser retrieveUsersWithIDs:[mutableUsers allObjects] withCompletion:^(NSArray<PFUser *> *userObjects) {
+            self.usersCheckedIn = userObjects;
+            long usersCount = self.usersCheckedIn.count - 2;
```

Comment:
I might give this a more specific name, and not define it until your "else" (right above                 self.usersLabel.text = [NSString stringWithFormat:@"%@, %@, and %ld have checked in here.", self.usersCheckedIn[0].displayName, self.usersCheckedIn[1].displayName, usersCount];) since we're only using it there